src/opm/input/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp

```diff
@@ -930,6 +1004,8 @@ namespace {
                 return ph.active(::Opm::Phase::GAS)
                     ? Opm::fun::map( famII_3p, Opm::fun::iota( num_tables ) )
                     : Opm::fun::map( famII_2p, Opm::fun::iota( num_tables ) );
+            case SatfuncFamily::III:
+                throw std::domain_error("Saturation keyword family III is not applicable for a oil systems");
```

Comment:
Tiny nit: We should not mix singular and plural in the diagnostic message ("A .. systemS").  We either need to say "..for AN oil system" (singular) or "... for oil systems" (plural).